packages/editor/src/components/post-featured-image/menu.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import { NavigableMenu, MenuGroup, MenuItem } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import {
+	media as mediaIcon,
+	upload as uploadIcon,
+	trash as trashIcon,
+} from '@wordpress/icons';
+
+export default function PostFeaturedImageMenu( {
+	removeImageLabel,
+	onClose,
+	onOpenMediaLibrary,
+	onOpenFileDialog,
+	onRemoveImage,
+} ) {
+	return (
+		<NavigableMenu className="editor-post-featured-image__menu">
```

Comment:
Should we use `MediaReplaceFlow` here? The icons are also at the right there and here are on the left.

Reply:
Not using a dropdown anymore!